client/src/pages/Profile.jsx

```diff
@@ -80,6 +80,52 @@ const Profile = () => {
         getProfileData(email)
     }, [])
 
+    const [getemail, setEmail] = useState('')
+
+    const handle = async () => {
+        // Getting user id from localstorage
+        const id = JSON.parse(localStorage.getItem('auth')).loginId;
+        console.log(id);
+        // Checking if the email is valid using regex
+        const emailRegex = /\S+@\S+\.\S+/;
+        if (!emailRegex.test(getemail)) {
+          // Checking if the user is an anonymous user
+          const auth = JSON.parse(localStorage.getItem('auth'));
+          if (auth.loginType === 'anonymous') {
+            // Handling anonymous user login 
+            const data = {
+              email: getemail,
+              loginId: id,
+            };
+            try {
+              const response = await api.post('/login', data);
```

Comment:
this is wrong, you didn't follow what i said at all

Reply:
I didn't understand, I have realized that this is beyond my knowledge. You can remove me from this one.

Reply:
yeah i figured, was just waiting for you to admit it, but no worries. Thanks for wanting to contribute, you can check out Simpler issues or use the app and raise an issue.